pkg/build-agent/file_server.py

```diff
@@ -3,25 +3,25 @@
 import time
 import threading
 
-import BaseHTTPServer
-import SimpleHTTPServer
+import http.server
+import http.server
 
 watched_files = ['SmartVA-Analyze.exe'.lower()]
 start = time.time()
 run_time = 60
 
 
-class SimpleHTTPRequestHandlerFileWatcher(SimpleHTTPServer.SimpleHTTPRequestHandler):
+class SimpleHTTPRequestHandlerFileWatcher(http.server.SimpleHTTPRequestHandler):
     def copyfile(self, source, outputfile):
         if isinstance(source, file):
             filename = os.path.split(source.name)[-1]
             if filename.lower() in watched_files:
                 watched_files.remove(filename.lower())
-        SimpleHTTPServer.SimpleHTTPRequestHandler.copyfile(self, source, outputfile)
+        http.server.SimpleHTTPRequestHandler.copyfile(self, source, outputfile)
 
 
-httpd = BaseHTTPServer.HTTPServer(('0.0.0.0', 8000), SimpleHTTPRequestHandlerFileWatcher)
-print('{address[0]}:{address[1]} - - Listening...'.format(address=httpd.server_address))
+httpd = http.server.HTTPServer(('0.0.0.0', 8000), SimpleHTTPRequestHandlerFileWatcher)
+print(('{address[0]}:{address[1]} - - Listening...'.format(address=httpd.server_address)))
```

Comment:
here and below, consider using f-strings as in:
`print(f'{httpd.server_address[0]}:{httpd.server_address[1]} - - Listening...')`